core/src/main/java/bio/terra/pearl/core/service/participant/search/facets/AnswerFacetValue.java

```diff
@@ -0,0 +1,67 @@
+package bio.terra.pearl.core.service.participant.search.facets;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+
+/**
+ * Much of this class mimics some of the type processing found in Answer.java.  However, we don't support querying
+ * objectValue stuff yet.
```

Comment:
Is this going to nudge us to convert the `objectValue` column to `jsonb`?

Reply:
yeah, probably